test/fork/add-asset-cbbtc.js

```diff
@@ -93,7 +93,6 @@ describe('add cbBTC asset to Pool', function () {
     this.proposalCategory = await ethers.getContractAt(abis.ProposalCategory, addresses.ProposalCategory);
     this.stakingPoolFactory = await ethers.getContractAt(abis.StakingPoolFactory, addresses.StakingPoolFactory);
     this.pooledStaking = await ethers.getContractAt(abis.LegacyPooledStaking, addresses.LegacyPooledStaking);
-    this.yieldTokenIncidents = await ethers.getContractAt(abis.YieldTokenIncidents, addresses.YieldTokenIncidents);
```

Comment:
there was no need to touch any fork tests except the `basic-functionality-tests.js`. they're one-offs, once upgraded - the test's only purpose is to serve as a template/example, nothing more. leave as is already :)